lldb/include/lldb/lldb-enumerations.h

```diff
@@ -819,6 +823,7 @@ enum BasicType {
   eBasicTypeFloat,
   eBasicTypeDouble,
   eBasicTypeLongDouble,
+  eBasicTypeFloat128,
```

Comment:
The same goes here, although apparently https://reviews.llvm.org/D120690 has snuck through and nobody noticed.